File: DotNet/Shared/Application/Models/Responses/ReportScheduleSummaryModel.cs
namespace LantanaGroup.Link.Shared.Application.Models.Responses
{
    public class ReportScheduleSummaryModel
    {
        public string ReportId { get; set; } = string.Empty;
        public string FacilityId { get; set; } = string.Empty;
        public DateTime StartDate { get; set; } = default;
        public DateTime EndDate { get; set; } = default;
        public DateTime? SubmitReportDateTime { get; set; }
        public List<string> Measures { get; set; } = [];
    }
}

File: Java/shared/src/main/java/com/lantanagroup/link/shared/entities/ReportScheduleSummaryModel.java
package com.lantanagroup.link.shared.entities;

import lombok.Getter;
import lombok.Setter;

import java.util.Date;
import java.util.List;

@Getter
@Setter
public class ReportScheduleSummaryModel {
    private String reportId;
    private String facilityId;
    private Date startDate;
    private Date endDate;
    private Date submitReportDateTime;
    private List<String> measures;
    private String payloadRootUri;
}
